pennylane_lightning/src/simulators/lightning_qubit/gates/CMakeLists.txt

```
@@ -15,48 +15,48 @@ if (ENABLE_GATE_DISPATCHER AND UNIX AND (${CMAKE_SYSTEM_PROCESSOR} MATCHES "(AMD
                             AssignKernelMap_AVX2.cpp
                             AssignKernelMap_AVX512.cpp
                             AssignKernelMap_Default.cpp CACHE INTERNAL "" FORCE)
-    add_library(lightning_qubit_gates_kernel_map STATIC ${KERNEL_MAP_FILES})
-    target_link_libraries(lightning_qubit_gates_kernel_map PRIVATE  lightning_qubit_utils lightning_external_libs lightning_compile_options)
-    set_target_properties(lightning_qubit_gates_kernel_map PROPERTIES POSITION_INDEPENDENT_CODE ON)
+    add_library(lqg_kernel_map STATIC ${KERNEL_MAP_FILES})
```

```suggestion
    add_library(lq_kernel_map STATIC ${KERNEL_MAP_FILES})
```
I think it'd be better to keep the namespace separated from the rest. Maybe we can think of a better and more succinct name here for the support (not sure if I like `lq_kernel_map` since it may obscure what is included if an external component uses this lib). Though, if we are shortening names for this purpose, it'd likely be a good call to add a README.md and explain that name -> support mapping, since it's going to be a little harder to reason out what is included from a given name.

I'd suggest making the changes in all `lqg` libs here and below.

---

Trying to avoid to over complicate it, I will test if we can make lq_gates_kernel_map first.
It will probably be more clear that lq refers to lightning_qubit.